demo.md

```diff
@@ -0,0 +1,44 @@
+# Setup
+
+### Starting the Node
+```
+rm -rf demo.db
+build/bin/vast -d demo.db start
```

Comment:
```suggestion
vast -d demo.db start
```

We should always assume that `$PATH` has been configured properly. Otherwise we run in cases that do not work with everybody.

(I'm only commenting once, but there are many occurrences of this.)